qutebrowser/misc/sessions.py

```diff
@@ -289,10 +307,6 @@ def save(self, name, last_window=False, load_next_time=False,
         Return:
             The name of the saved session.
         """
-        name = self._get_session_name(name)
```

Comment:
This is a behavior change, I don't think you can remove it.

Reply:
I can fix this... but is this a review to finally merge the PR? otherwise, i can resume working on when it is time.

Reply:
I'm trying to slowly coallate as many PRs as possible, as I think qutebrowser master has some pretty nasty issues. If it diverges enough (or if there are other issues), I might fork entirely. If it's fixed, I'll add this PR to the whitelist.

https://gitlab.com/jgkamat/dotfiles/blob/master/qutebrowser/gen_integration.sh

Reply:
Hey @jgkamat, i checked your concern about removing this line. It's not actually removed, i just moved it to `_save_session_file` [here](https://github.com/qutebrowser/qutebrowser/pull/4346/files#diff-5537465496c0c5598cfe8b031fa6d74aR418)

Reply:
The origjnal (not the mutated) name variable is used later on in this function, whereas before the mutated name was used everywhere.